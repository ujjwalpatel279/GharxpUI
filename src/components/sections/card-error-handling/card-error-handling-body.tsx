import React, { FunctionComponent, ReactElement } from 'react';
import { CardBody } from '../../shared/card/card-body';

import './card-error-handling.scss';

export const CardErrorHandlingBody: FunctionComponent = (): ReactElement => {
  return (
    <CardBody>
      <div className="s-card-bad-redirects__detail">
        <div className="s-card-bad-redirects__detail__content">
          <b>Something went wrong !</b>
        </div>
      </div>
      <div className="s-card-bad-redirects__detail">
        <div className="c-cart__detail__content">
          <p>something went wrong and we couldn&apos;t complete your request. please refresh the page to try again</p>{' '}
        </div>
      </div>
    </CardBody>
  );
};
