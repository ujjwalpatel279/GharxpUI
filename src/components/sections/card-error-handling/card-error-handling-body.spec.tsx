import React from 'react';
import { render, cleanup } from '@testing-library/react';
import { CardErrorHandlingBody } from './card-error-handling-body';

describe('shows card bad redirects body', () => {
  afterEach(cleanup);
  it('show bad redirects information on card', () => {
    const { asFragment } = render(<CardErrorHandlingBody />);
    expect(asFragment()).toMatchSnapshot();
  });
});
