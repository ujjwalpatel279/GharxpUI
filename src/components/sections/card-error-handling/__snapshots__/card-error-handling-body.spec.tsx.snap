// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shows card bad redirects body show bad redirects information on card 1`] = `
<DocumentFragment>
  <div
    class="c-card__body"
  >
    <div
      class="s-card-bad-redirects__detail"
    >
      <div
        class="s-card-bad-redirects__detail__content"
      >
        <b>
          Something went wrong !
        </b>
      </div>
    </div>
    <div
      class="s-card-bad-redirects__detail"
    >
      <div
        class="c-cart__detail__content"
      >
        <p>
          something went wrong and we couldn't complete your request. please refresh the page to try again
        </p>
         
      </div>
    </div>
  </div>
</DocumentFragment>
`;
